dev/cloud/naming_conventions.md

```
@@ -0,0 +1,259 @@
+
+In case multiple versions exist of the same service with seperate databases the service version should be added at the end.
+```
+simplic_oxs_staging_logistics_v3
+```
+
+
+#### Collection name
+
+The collection name should __*ALWAYS*__ be loweercase.
+Collection names are __*FORBIDDEN*__ to contain the following characters:
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)